synthtool/languages/node.py

```diff
@@ -317,13 +317,11 @@ def owlbot_main(
             s_copy([library], excludes=staging_excludes)
         # The staging directory should never be merged into the main branch.
         shutil.rmtree(staging)
-    else:
+    elif default_version:
```

Comment:
@SurferJeffAtGoogle if we want to keep the logic of grabbing versions from `src/`, I think this compromise would get us back to the existing behavior, while allowing `default_version` to be optional.